yang/frr-route-map.yang

```diff
@@ -239,20 +252,23 @@ module frr-route-map {
                 type string;
```

Comment:
Suggestion: since we are going to use leaf-refs next version, would you mind turning this string into a frr-interface leaf ref again?

Reply:
@rzalamena  once the libyang1.x is available in frr CI process, we will move all required places from `string` to `leafref`.